frontend/src/app/components/ExperimentPage.js

```diff
@@ -18,7 +18,25 @@ import ExperimentPreFeedbackDialog from '../components/ExperimentPreFeedbackDial
 
 const CHANGE_HEADER_ON = 105;
 
+
 export default class ExperimentPage extends React.Component {
+  render() {
+    const { getExperimentBySlug, params } = this.props;
+    const experiment = getExperimentBySlug(params.slug);
```

Comment:
Also considering at some point we'll use this component in static rendering, easier to just pass in an experiment than a lookup-helper and a param.slug